startup.sh

```diff
@@ -1,11 +1,11 @@
 if [ $# -eq 0 ] ; then
   echo '=========== Precompiling'
-  cd spec/dummy; RAILS_ENV=production SECRET_KEY_BASE=does_not_matter_here bin/rails assets:precompile
+  cd spec/dummy; RAILS_ENV=production SECRET_KEY_BASE=does_not_matter_here bin/rails assets:precompile --trace
```

Comment:
This is another one that I will need to test out. Don't think it will cause any issues though.

Reply:
Sorry, I don't think we need this, I was trying to see CI errors but I think the `webpack_compile_output ` is enough, let me remove it.

Reply:
Done